.github/workflows/libkineto_cuda.yml

```diff
@@ -0,0 +1,77 @@
+name: libkineto PR Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+# Use TorchBench's docker image which has all basic dependencies.
+env:
+  CONDA_ENV: "torchbench"
+  DOCKER_IMAGE: "ghcr.io/pytorch/torchbench:latest"
+  SETUP_SCRIPT: "/workspace/setup_instance.sh"
+
+jobs:
+  pr-test:
+    # AWS linux instance label: c5.2xlarge
+    runs-on: [linux.2xlarge]
+    timeout-minutes: 180 # 3 hours
+    environment: docker-s3-upload
```

Comment:
Need to remove this line - this is only needed if you need to access github secrets in this workflow